gemoc_studio/releng/org.eclipse.gemoc.gemoc_studio.product/gemoc_studio.product

```diff
@@ -73,6 +74,16 @@
       <plugin id="org.eclipse.update.configurator" autoStart="true" startLevel="3" />
    </configurations>
 
+   <repositories>
+      <repository location="http://timesquare.inria.fr/ecl/update_site/neon" enabled="false" />
+      <repository location="http://timesquare.inria.fr/update_site/neon" enabled="false" />
+      <repository location="http://melange-lang.org/updatesite/" enabled="true" />
+      <repository location="http://archive.eclipse.org/nebula/Q22016/incubation" enabled="true" />
+      <repository location="http://download.eclipse.org/releases/oxygen" enabled="true" />
+      <repository location="http://www.kermeta.org/k3/update/" enabled="true" />
+      <repository location="http://ci.eclipse.org/gemoc/job/gemoc-studio/job/eclipse-oxygen/lastSuccessfulBuild/artifact/gemoc-studio/gemoc_studio/releng/org.eclipse.gemoc.gemoc_studio.updatesite/target/repository/" enabled="true" />
```

Comment:
I think you put the update site for the `eclipse-oxygen` branch. Shouldn't it be https://ci.eclipse.org/gemoc/job/gemoc-studio/job/master/lastSuccessfulBuild/artifact/gemoc-studio/gemoc_studio/releng/org.eclipse.gemoc.gemoc_studio.updatesite/target/repository/ instead?

Reply:
right I'll do that now ...